src/content/contributing/style-guide/index.md

```diff
@@ -0,0 +1,200 @@
+
+#### Enterprises
+
+People, businesses, and other organizations who want to understand Ethereum's value in an enterprise setting
+
+Because ethereum.org caters to a wide variety of users, all content on the site should make an effort to explain any technologies and concepts as simply as possible.
+
+## Objectivity {#objectivity}
+
+Ethereum.org documentation aims to maintain a credibly neutral source of truth to inform readers about Ethereum and its ecosystem. Some examples of things that we don't want in the content on ethereum.org:
+
+**Grand, unverifiable claims about Ethereum or adjacent technologies**
+
+> e.g. _ "Ethereum will take over the world because..."_
+
+**Hostile or confrontational language aimed at any organization or person**
+
+> e.g. _ "Company X is bad because they are centralized!"_
+
+**Politically charged rhetoric**
+
+> e.g. _ "This political party is better for decentralization because..."_
+
+## Tone {#tone}
+
+The tone of the content on the site should be welcoming, friendly and authoritative. Where possible, jargon should be minimized and simple language used instead.
+
+## Best Practices {#best-practices}
+
+### Use American English {#american-english}
+
+For words that have multiple spellings, use American English over British English.
+
+**For example:**
+
+- `decentralized` over `decentralised`
+- `color` over `colour`
+- `analyze` over `analyse`
+
+### Acronyms {#acronyms}
+
+When introducing an unfamiliar acronym, spell out the full term, and put the acronym in parentheses. Put both the full term and acronym in bold.
+
+**For example:**
+
+"Ethereum, like Bitcoin, currently uses a consensus protocol called **Proof-of-work (PoW)**."
+
+### Consistency {#consistency}
+
+Many of the topics covered on [Ethereum.org](http://ethereum.org) are technically complex. To reduce confusion to the reader, terms should be used consistently. For example, don't cycle back-and-forth between proof-of-work and PoW at random.
+
+### Ethereum {#ethereum}
+
+Ethereum is a proper noun and should always be capitalized.
+
+- `Ethereum` not `ethereum`
+
+### Ether {#ether}
+
+Ether is a common noun and should not be capitalized unless at the beginning of a sentence. ETH, on the other hand, is a currency abbreviation and should always be capitalized.
+
+- `ether` not `Ether`
+- `ETH` not `eth or Eth`
+
+### Mainnet {#mainnet}
+
+When referring to the Ethereum Mainnet (i.e. not referring to a testnet) use the proper noun. Proper nouns help avoid confusion and build greater understanding.
+
+**Correct usage:**
+
+- Mainnet
+- Ethereum Mainnet
+
+**Incorrect usage:**
+
+- main net
+- mainnet
+- Main net
+- Ethereum mainnet
+
+### Proof-of-work / Proof-of-stake {#proof-of}
+
+Proof-of-work should be capitalized at the beginning of a sentence or when the term is introduced for the first time on a page, in any other instance, all letters should be lower case. In either case, proof-of-work should be hyphenated between each word.
```

Comment:
> or when the term is introduced for the first time on a page

Should we be capitalizing this mid-sentence ever? I don't think capitalizing terms that are new to a page is a convention we've done elsewhere.

Reply:
Also, perhaps we should just broaden this to proof-of-whatever.... show the PoW examples, but note that would similarly apply to proof-of-stake, proof-of-humanity, proof-of-individuality, proof-of-authority, etc.

Reply:
Agreed, I don't think capitalizing mid-sentence is necessary.